test/fail/par-bad-base.mo

```diff
@@ -0,0 +1,19 @@
+class migration({}) = {};
+
+(with migration; moot = "MOOT")
+actor {
+  func a() : async () {
+    await ({ cycles = -3 } with) async ();
+    await ({ cycles = "" } with) async ();
+    await (with cycles = "") async ();
+    func cycles() {};
+    await (with cycles) async ();
+    await (with timeout = "") async ();
+  };
+  func b() : async () {
+    object base {
+      public func cycles() {}
+    };
+    await (base with) async ();
```

Comment:
Hmm, this is kinda ugly.
I wonder if
(with object-combination) would work in the parser...
(with base)
(with cycles = 100l, ...)
(with base with cycles = 100).
Probably not because of punning...